TraceScreen.h

```diff
@@ -18,9 +18,9 @@ in the source distribution for its full text.
 
 typedef struct TraceScreen_ {
    InfoScreen super;
-   bool tracing;
    pid_t child;
```

Comment:
Depending on the system it's likely to have `pid_t` might not necessarily be 64 bit, but the `FILE *`  will be. Thus placing the `FILE*` before the child's PID can have some advantages from a layout PoV.